docker/prod/Dockerfile

```diff
@@ -18,7 +18,7 @@ ENV APP_PATH=/app
 ENV APP_DATA_PATH=/var/openproject/assets
 ENV PGVERSION="13"
 ENV CURRENT_PGVERSION="13"
-ENV NEXT_PGVERSION="15"
+ENV NEXT_PGVERSION="17"
```

Comment:
Use ARG instead maybe, to make it clear that this is a build-time only variable.